benchmarks/src/test/resources/create-datasource-and-dashboard.sh

```diff
@@ -13,10 +13,9 @@
 # the License.
 #
 
-
 set -xeuo pipefail
 
-if ! curl --retry 5 --retry-connrefused --retry-delay 0 -sf http://grafana:3000/api/dashboards/name/prom; then
+if ! curl --retry 5 --retry-connrefused --retry-delay 0 -sf http://grafana:3000/api/datasources/name/prom; then
     curl -sf -X POST -H "Content-Type: application/json" \
          --data-binary '{"name":"prom","type":"prometheus","url":"http://prometheus:9090","access":"proxy","isDefault":true}' \
          http://grafana:3000/api/datasources
```

Comment:
this was a pre-flight check that checked the wrong condition, but wouldn't have mattered